gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -249,8 +249,10 @@
 
   private static final String KNOX_TOKEN_EVICTION_INTERVAL = GATEWAY_CONFIG_FILE_PREFIX + ".knox.token.eviction.interval";
   private static final String KNOX_TOKEN_EVICTION_GRACE_PERIOD = GATEWAY_CONFIG_FILE_PREFIX + ".knox.token.eviction.grace.period";
+  private static final String KNOX_TOKEN_PERMISSIVE_FAILURE_ENABLED = GATEWAY_CONFIG_FILE_PREFIX + ".knox.token.permissive.failure.enabled";
```

Comment:
I don't love this config property name. Perhaps, **.knox.token.permissive** is sufficient ? Or, **.knox.token.permissive.validation** ?

Reply:
Sure, I'll change it to `.knox.token.permissive.validation`